src/Directory.Build.props

```diff
@@ -11,4 +11,8 @@
     <PackageReference Include="Internal.AspNetCore.Sdk" PrivateAssets="All" Version="$(InternalAspNetCoreSdkPackageVersion)" />
     <PackageReference Include="Internal.AspNetCore.Analyzers" PrivateAssets="All" Version="$(InternalAspNetCoreAnalyzersPackageVersion)" />
   </ItemGroup>
-</Project>
+
+  <!--<PropertyGroup>
+    <EnableApiCheck>false</EnableApiCheck>
+  </PropertyGroup>-->

```

Comment:
What's this about? Bad merge or something intentional?

Reply:
I was running into an issue with it for some reason a little while ago but it's gone now.. 
Removing this